### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by techEmprez**
### **I implemented the following in this PR**

-  For each URL you should create:
    - A route.
    - An action in the correct controller.
    - A view file.

-  View files should include only placeholders written in plain HTML, e.g:
    - Here is a list of posts for a given user

-  Make sure that you added code to handle **only** the 4 required URLs.

###### Make sure that when you run the `rails routes` command in your console, you can see the following output:
Single Post page wireframe

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1,4 @@
+feature4/controllers
```
Comment (Davidosky007) [inline .ruby-version:1]: * In the requirements you are required not to use rails generate scaffold command. The point of this project is to show you how to set up controllers by yourself.
* View files should include only placeholders written in plain HTML,
* Make sure that you added code to handle only the 4 required URLs, Kindly stick to all these requirements as currently, your code doesn't follow the requirements 
image
image
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @techEmprez 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Well written PR 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .ruby-version (hunk 1 above)
Comment (techEmprez) [reply .ruby-version:1]: All changes have been implemented

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @techEmprez,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**